packages/blade/.babelrc.js

```diff
@@ -8,6 +8,7 @@ const configs = {
           {
             alias: {
               '@storybook/react': '@storybook/react-native',
+              '^styled-components$': 'styled-components/native',
```

Comment:
I see you've already done this here 😅 

Styled components recommend this approach: https://styled-components.com/docs/basics#simpler-usage-with-the-metro-bundler

#449

Reply:
yeah but I'm not doing it for all the files. I'm basically doing where react-native will refer to a common component(common for web and native). rest at all the places we shall explicitly write `/native` since there are many places jaha pe types also fail since they are diff for web and native :()